bucket/freeplane.json

```diff
@@ -5,8 +5,8 @@
     "license": "GPL-2.0-or-later",
     "suggest": {
         "JDK": [
-            "java/oraclejdk",
-            "java/openjdk"
+            "java/oraclejdk17",
```

Comment:
`java/oraclejdk17` does not exist, remove this item or replace it with `java/oraclejdk-lts`.

Reply:
OR replace them with `java/openjdk11` and `java/openjdk17`, then user will know that it can run in Java 11 to 17 without view the documentation.